subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -1223,7 +1223,7 @@ Existing clients should migrate to the <<publishing_maven.adoc#publishing_maven,
 The concept of conventions is outdated and superseded by extensions. To reflect this in the Gradle API, the following elements are now deprecated:
 
 - link:{javadocPath}/org/gradle/api/Project.html#getConvention--[Project.html#getConvention()]
-- link:{javadocPath}/org/gradle/api/internal/HasConvention.html[HasConvention]
+- `org.gradle.api.internal.HasConvention` (deprecated)
```

Comment:
How come we've removed this link? Do we not generate javadoc for deprecated types?

Reply:
We do not, it's a dead link.